ports/psoc6/modules/machine/modmachine.h

```diff
@@ -1,21 +1,30 @@
 #ifndef MICROPY_INCLUDED_PSOC6_MODMACHINE_H
 #define MICROPY_INCLUDED_PSOC6_MODMACHINE_H
 
-#include "py/obj.h"
+// std includes
+#include <stdio.h>
 
+// mpy includes
+#include "py/obj.h"
```

Comment:
These includes are not used in the header, so leave it in the .c file

Reply:
Reworked all includes. All the includes are in the locations/files where there are actually used.